GSA_Adapter/Convert/FromGsa/Loads/NodeLoad.cs

```diff
@@ -0,0 +1,146 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using BH.oM.Structure.Loads;
+using System;
+using BH.Engine.Adapter;
+using BH.oM.Adapters.GSA;
+using System.Collections.Generic;
+using BH.oM.Structure.Elements;
+using BH.oM.Geometry;
+using BH.oM.Base;
+using System.Linq;
+
+
+namespace BH.Adapter.GSA
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static PointLoad FromGsaNodeLoad(string gsaString, Dictionary<string, Loadcase> lCases, Dictionary<string, Node> nodes, double unitFactor)
+        {
+            if (string.IsNullOrWhiteSpace(gsaString))
+                return null;
+
+            string[] gStr = gsaString.Split(',');
+
+            if (gStr.Length < 7)
+                return null;
+
+            string lCaseNo = gStr[3];
```

Comment:
Can we include a comment with an example string, so anyone debugging later knows what the string was expected to look like when you wrote this code @johannaisak ? 😄

Reply:
Done :)